tidyblocks/tidyblocks.js

```diff
@@ -349,6 +349,56 @@ class TbDataFrame {
     return this
   }
 
+  ttestPlot (blockId, environment, testFunc, parameters, ...columns) {
+    const {result, legend} = testFunc(blockId, this, parameters, columns)
+
+    const L1 = -1*Math.abs(result.statistic)
```

Comment:
what do the `1` and `2` mean?

Reply:
I changed this code to be more human readable and will probably try to make it prettier in functions anyways

Reply:
you're a good person - lemme know when you want it merged, or just go ahead and merge it yourself